doc/testing_samples/samples_list.rst

```diff
@@ -1,14 +1,68 @@
+      |                             |                                                                          |                                               |                               |                       |              |             |             |
+      +-----------------------------+--------------------------------------------------------------------------+-----------------------------------------------+-------------------------------+-----------------------+--------------+-------------+-------------+
+      | :ref:`template_subghz`      | * External flash (included in the DK)                                    | * MCUboot - DFU partition in external flash   | * :file:`fsk.conf`            | --                    | Supported    | Supported   | Supported   |
+      |                             | * `Semtech SX1262MB2CAS`_ (only for the configurations with LoRa or FSK) | * Application partition size = 956 kB         |                               |                       |              |             |             |
+      |                             |                                                                          |                                               |                               |                       |              |             |             |
+      +-----------------------------+--------------------------------------------------------------------------+-----------------------------------------------+-------------------------------+-----------------------+--------------+-------------+-------------+
+      | :ref:`template_ble`         | --                                                                       | * MCUboot - DFU partition in internal flash   | --                            | Supported             | --           | --          | Supported   |
+      |                             |                                                                          | * Application partition size = 480 kB         |                               |                       |              |             |             |
+      +-----------------------------+--------------------------------------------------------------------------+-----------------------------------------------+-------------------------------+-----------------------+--------------+-------------+-------------+
+      | :ref:`device_under_test`    | * `Semtech SX1262MB2CAS`_ (optional)                                     | --                                            | --                            | Supported             | Supported    | Supported   | --          |
+      +-----------------------------+--------------------------------------------------------------------------+-----------------------------------------------+-------------------------------+-----------------------+--------------+-------------+-------------+
+
+   .. tab:: nRF5340*
+
+      **TBU FOR NRF5340**

```

Comment:
Information for nRF5340 needed

Reply:
all nRF5340 support is experimental for now. But I think this table can be the same like for nrf52

Reply:
got it - left the table the same as in case of nrf52